view/src/extras/surplusStepper.js

```diff
@@ -0,0 +1,217 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Stepper from "@material-ui/core/Stepper";
+import Step from "@material-ui/core/Step";
+import StepLabel from "@material-ui/core/StepLabel";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import FarmForm from "../extras/farmForm";
+import ProduceForm from "../extras/produceForm";
+import SurplusForm from "../extras/surplusForm";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+  },
+  button: {
+    marginRight: theme.spacing(1),
+    color: theme.palette.primary.contrastText,
+    backgroundColor: theme.palette.primary.dark,
+  },
+  instructions: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+/** Defines the steps in the Stepper (each is a form) */
+function getSteps() {
+  return [
+    "Create a Farm",
+    "Create a Produce Object",
+    "Create a Surplus Object",
+  ];
+}
+
+export default function SurplusStepper(props) {
+  const classes = useStyles();
+  // stepper states
+  const [activeStep, setActiveStep] = React.useState(0);
+  const [skipped, setSkipped] = React.useState(new Set());
+  // form states (relevant to the final step - surplus form)
+  const [currentFarm, setCurrentFarm] = React.useState("");
```

Comment:
Seems odd that the initial value for this is a string.  Is this really the farm state, or is this just the farm ID?  (If the latter, I recommend renaming this to make that more obvious.)

Reply:
It actually is an entire farm object.... So I switched the default state to {} so that it is more clear that it is loading a JSON file